apps/firelens-stability/collections/ecs-firelens-stability-tests/s3-stability/cases/s3-throughput-30mbps.json

```diff
@@ -0,0 +1,5 @@
+{
+    "definitions": {
+        "datajet_kbps_throughput_tcp": "30000"
+    }
+}
```

Comment:
Possibly best to spread the output among multiple s3 outputs.
- Use multiple inputs or multiple tags
   - Tcp -> tcp1 -> 10mb/s
   - Tcp -> tcp2 -> 10mb/s
   - Tcp -> tcp3 -> 10mb/s

- One S3 Output

Reply:
This is done.